lib/AST/ASTmisc.cpp

```diff
@@ -267,14 +263,9 @@ void FatalError(const char* str)
   if (vc_error_hdlr)
   {
     vc_error_hdlr(str);
-    assert(false);
-    exit(-1);
-  }
-  else
-  {
-    assert(false);
-    exit(-1);
   }
+  assert(false);
+  exit(-1);
```

Comment:
I think we should change this to ``abort()``. If debugging with gdb, gdb will catch an ``abort()`` but not a call to ``exit()``.